frontend/README.md

```diff
@@ -0,0 +1,49 @@
+# Time Distance Standards (Frontend) [![Build Status][build]](https://circleci.com/gh/bayesimpact/tds) [![apache2]](https://www.apache.org/licenses/LICENSE-2.0)
+
+[build]: https://img.shields.io/circleci/project/bayesimpact/tds.svg?branch=master&style=flat-square
+[apache2]: https://img.shields.io/badge/License-Apache%202.0-blue.svg
+
+> Explore Healthcare network adequacies and population distance to services.
+
+## Install
+
+Clone this repo:
+
+```sh
+git clone git@github.com:bayesimpact/tds.git
+```
+
+## Build & Run
+
+```sh
+docker build -t tds/frontend .
```

Comment:
you prefer to not use docker-compose?

Reply:
The docker-compose.yml in the root directory - what would this look like in terms of `docker-compose`?